### DISCUSSION TITLE
Prisma Client API returns entire timestamp for @db.Date or @db.Time

### DISCUSSION THREAD

**Discussion by WTFace** [prisma/prisma#16410] [Q&A]
Hi. If I save any model using @db.Time or @db.Date I can see only time and date in pgAdmin.
Using prisma.model.findMany(), those fields are entire timestamp in ISO format.

For example, if I convert '00:10' to ISO and save it to a field with @db.Time, I see 
- 00:10:00 in pgAdmin. 
- '1970-01-01T00:10:00.000Z' in Prisma Client API return value

**Comment by pantharshit00** [comment]
Hello @WTFace 

I think this is the expected behaviour here as Prisma converts all `Date` type fields to a JS Date. 

You can open a feature request if you want any alterations in this behaviour. You can open it here: https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
